File: Atcoder_#350/ruby/b_dentist_aoki.rb
n, q = gets.split.map(&:to_i)
treatments = gets.split.map(&:to_i)

teeth = Array.new(n, true)

treatments.each do |t|
  index = t - 1
  teeth[index] = !teeth[index]
end

result = teeth.count(true)

puts result

File: Atcoder_#350/javascript/b_dentist_aoki.js
function main(input) {
  const lines = input.trim().split('\n');

  const [n, q] = lines[0].split(' ').map(Number);

  const treatments = lines[1].split(' ').map(Number);

  const teeth = Array(n).fill(true);

  treatments.forEach(t => {
      const index = t - 1;
      teeth[index] = !teeth[index];
  });

  const result = teeth.filter(Boolean).length;

  console.log(result);
}
main(require("fs").readFileSync("/dev/stdin", "utf8"));
